src/createNumbersGenerator.js

```diff
@@ -29,8 +29,17 @@
  * @param {number} min - (optional)
  * @param {number} max - (optional)
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+
+function createNumbersGenerator(numbers, min = -Infinity, max = Infinity) {
+  let i = 0;
+
+  const filtered = numbers.filter((unit) => {
+    return (unit >= min && unit <= max);
```

Comment:
не пиши скобки в return 
```suggestion
    return unit >= min && unit <= max;
```

Reply:
Гаразд, тут так вийшло, бо я скопіював його з попередньої версії.